maintainers/maintainer-list.nix

```diff
@@ -1,68 +1,68 @@
 /*
   List of NixOS maintainers.
-   ```nix
```

Comment:
Why?

Reply:
because the CI of the PR failed on the linter step and required running `nix-shell` && `nixfmt maintainers/maintainers-list.nix` (afterwards CI succeeded)

Reply:
Running `nixfmt` on the file from master, both before & after adding the attribute with your data, doesn't result in such a large diff for me.

Try rebasing on master, dropping these changes to this, re-adding your attribute, and committing it as a separate commit that precedes the package init: https://github.com/NixOS/nixpkgs/blob/773e1bd8f733a4ea63a247754e24cb5844cc3f30/CONTRIBUTING.md#commit-conventions

> When adding yourself as maintainer in the same pull request, make a separate commit with the message `maintainers: add <handle>`. Add the commit before those making changes to the package or module. See [Nixpkgs Maintainers](https://github.com/NixOS/nixpkgs/blob/773e1bd8f733a4ea63a247754e24cb5844cc3f30/maintainers/README.md) for details.